config/logging.php

```diff
@@ -5,7 +5,7 @@
         'name' => env('CLOUDWATCH_LOG_NAME', ''),
         'region' => env('CLOUDWATCH_LOG_REGION', ''),
         'credentials' => [
-            'key' => env('CLOUDWATCH_LOG_KEY', ''),
+            'key' => env('CLOUDWATCH_LOG_KEY', ''), // Leave empty if EC2 role has access to cloudwatch
```

Comment:
Please remove the comment here and add this to the readme. It should say not to define the env if to use the EC2 role itself.